requirements/system/ubuntu/apt-packages.txt

```diff
@@ -3,8 +3,9 @@ pkg-config
 gfortran
 libatlas-dev
 libblas-dev
+libgtk-3-0
```

Comment:
this requirement is necessary, solves this problem "libgtk-3.so.0: cannot open shared object file: No such file or directory"

Reply:
Something that we can do, is to build an image using the Dockerfile that is already on the repository and use that for the tests, or use https://hub.docker.com/r/openedx/edx-platform which is suppose to be created with that Dockerfile

Reply:
so, should i remove this requirement?

Reply:
Yes, if you need new apt requirements add them to the scripts/circle-ci-configuration.sh file.

Reply:
Ok, i'll test it

Reply:
Another suggestion, use a mongodb container instead of installing mongodb in the test runner:

Something like this:

https://github.com/eduNEXT/edx-platform/commit/83b9feeb90204c1f2b502084428e4f9ea86eafca

Also remove the usage of apt-packeges.txt

Reply:
What version of mongo should i use @morenol? 3.6 or 4.4?

Reply:
3.6 for juniper

Reply:
since this is master I would also asume 3.6